ush/mesoscale/evs_prepare_sref.sh

```diff
@@ -176,9 +176,6 @@ if [ $modnam = ccpa ] ; then
 
     ${METPLUS_PATH}/ush/run_metplus.py -c ${PARMevs}/metplus_config/machine.conf -c ${PRECIP_CONF}/PcpCombine_obsCCPA06h.conf
     export err=$?; err_chk
-    echo "Print PcpCombine_obsCCPA06h log files begin:"
-    cat $DATA/pb2nc/logs/*
-    echo "Print PcpCombine_obsCCPA06h log files end"
 
  else
```

Comment:
@BinbinZhou-NOAA 

You should not be deleting the cat'ting of the log files, because it is required to cat the log files.  But I think the problem is that you are pointing to the wrong directory.

Since you are running pcpcombine instead of pb2nc, you need to point to the correct directory here.  

Instead of $DATA/pb2nc/logs, the directory you should be using is:

$DATA/ccpa.$VDATE/logs

This the change you need to be making, not removing these statements.  

@AliciaBentley-NOAA @ShelleyMelchior-NOAA